src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/TupleTests.vb

```diff
@@ -0,0 +1,187 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.ExpressionEvaluator
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.UnitTests
+Imports Microsoft.VisualStudio.Debugger.Clr
+Imports Microsoft.VisualStudio.Debugger.Evaluation
+Imports System.Collections.Immutable
+Imports Xunit
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator.UnitTests
+
+    Public Class TupleTests
```

Comment:
No, I wasn't planning on porting more of the C# tests since most of the code in the ResultProvider is shared between C# and VB. I'll port more if we add tests that have language-specific behavior.